tensorboard/backend/event_processing/event_accumulator.py

```diff
@@ -80,16 +74,14 @@
 NORMAL_HISTOGRAM_BPS = (0, 668, 1587, 3085, 5000, 6915, 8413, 9332, 10000)
 
 DEFAULT_SIZE_GUIDANCE = {
-    COMPRESSED_HISTOGRAMS: 500,
     IMAGES: 4,
     AUDIO: 4,
     SCALARS: 10000,
-    HISTOGRAMS: 1,
-    TENSORS: 10,
+    HISTOGRAMS: 500,
+    TENSORS: 500,
```

Comment:
This is true. As discussed offline, I'll create a patch to fix that by preserving the status quo for now.